components/raftstore/src/store/peer.rs

```diff
@@ -2580,6 +2593,30 @@ where
         true
     }
 
+    fn notify_leader_the_peer_is_available<T: Transport>(
+        &mut self,
+        ctx: &mut PollContext<EK, ER, T>,
+    ) {
+        if !self.is_initialized() {
```

Comment:
What if it's still applying snapshot?

Reply:
This function will only be called in `CheckApplyingSnapStaus::Success` branch. So there will be no applying snapshots?